main.lua

```diff
@@ -2,22 +2,48 @@ require('mobdebug').start();             -- enable debugging checkpoints
 StartDebug();                            -- enable debugging
 local mod = RegisterMod("AI Final", 1);  -- register mod in game
 
---[[ uncomment this when we need to include config
-local _, err = pcall(require, "config")
-err = tostring(err)
-if not string.match(err, "attempt to call a nil value %(method 'ForceError'%)") then
+-----------------------------------
+-------- UTILITY FUNCTIONS --------
+-----------------------------------
+
+function import(filename)
+  local _, err = pcall(require, filename)
+  err = tostring(err)
+  if not string.match(err, "attempt to call a nil value %(method 'ForceError'%)") then
     if string.match(err, "true") then
         err = "Error: require passed in config"
     end
-    Isaac.DebugString(err)
-    print(err)
+    CPrint(err)
+  end
+end
+
+--[[ Write str to the Isaac Console --]]
+function CPrint(str) 
+  Isaac.ConsoleOutput(str.."\n")
+end
+
+--[[ Write str to file specified by config (or hardcoded) --]]
+function log(str)
+  -- TODO change this to write to a log file.
+  CPrint(str)
+end
+
+--[[ Utility method for testing --]]
+function equal(expected, result)
+  -- TODO make this smart?
+  return expected == result
+end
+
+--[[ Check membership of val in list --]]
```

Comment:
not a fan of the block comment look tbh, super clunky

Reply:
Just want to keep it consistent if there are multiline comments required for a function

Reply:
If there was any way to do multiline comments I would prefer:

--[[
does this thing
--]]

but I still think that single line comments look better